cpp/bench/ann/src/cuvs/cuvs_vamana.cu

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
```

Comment:
Please check the copyright years.
@cjnolet isn't this checked in style checks in CI?